src/injectIntl.tsx

```diff
@@ -9,6 +9,13 @@ export type ReactIntlPhraseProps = {
 
 export function injectIntl(WrappedComponent: React.ComponentType<ReactIntlPhraseProps>, options?: Parameters<typeof injectIntlReact>[1]): ReturnType<typeof injectIntlReact> & React.FC<ReactIntlPhraseProps> {
     class InjectPhrase extends React.Component implements ReactIntlPhraseProps {
+        constructor(props, context) {
+            super(props, context);
```

Comment:
```suggestion
        constructor (...args: ConstructorParameters<typeof React.Component>) {
            super(...args);
```

Reply:
I think writing it like this makes it more future-bulletproof in case of any changes in extended class's constructor

Reply:
Thanks for the suggestion, I removed the constructor altogether in 4010d61.